lpa-submissions-e2e-tests/cypress/integration/accuracyAppellantsSubmission.feature

```diff
@@ -0,0 +1,70 @@
+Feature: Accuracy of the Apellants Submission
```

Comment:
@wip tag needs to be added as the functionality is still not available and it will fail the tests on pipeline.

Reply:
That's the idea. It is supposed to fail at this stage until dev work is complete. This avoids any issues with tests not running that should be etc.